pilot/pkg/model/config.go

```diff
@@ -885,6 +903,22 @@ func (store *istioConfigStore) ServiceRoleBindings(namespace string) []Config {
 	return bindings
 }
 
+func (store *istioConfigStore) MeshRbacConfig() *Config {
+	meshRbacConfig, err := store.List(MeshRbacConfig.Type, "")
+	if err != nil {
+		log.Errorf("failed to get meshRbacConfig: %v", err)
+	}
+	if len(meshRbacConfig) > 1 {
+		log.Errorf("found %d meshRbacConfig, expecting only 1.", len(meshRbacConfig))
+	}
+	for _, rc := range meshRbacConfig {
+		if rc.Name == DefaultRbacConfigName {
+			return &rc
+		}
+	}
+	return nil
+}
+
 func (store *istioConfigStore) RbacConfig() *Config {
 	rbacConfigs, err := store.List(RbacConfig.Type, "")
 	if err != nil {
```

Comment:
This should no longer be an error log. If MeshRbacConfig is used, we do not expect RbacConfig exist. Maybe change it to an "info" log.  

Also, if users still use RbacConfig, it is better to raise a warning that "RbacConfig is deprecated. Use MeshRbacConfig instead". The best place is in the validation code when users enters an "RbacConfig".

Reply:
Updated. I logged the warning as the validate framework doesn't support to print a warning message to user.
Later we may change it to allow printing a warning message.